installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned.adoc

```diff
@@ -11,6 +11,7 @@ VMware vSphere instance by using installer-provisioned infrastructure.
 
 include::snippets/vcenter-support.adoc[]
 
+[id="prerequisites_installing-vsphere-installer-provisioned_{context}"]
```

Comment:
Is the `_{context}` needed for an assembly? Looking at the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies), I'm not sure it's needed here but I could be missing the reason.

Reply:
Yes, sorry. I followed Mike's style.